spec/features/service_providers_spec.rb

```diff
@@ -96,3 +95,29 @@
     expect(page).to have_content('Success')
   end
 end
+
+feature 'User Group is set based on the users user group' do
```

Comment:
nice tests! I like to keep feature specs to one feature per file. Put these in their own file? or incorporate into another feature spec as sub-scenarios?